guides/v2.2/cloud/project/project-conf-files_magento-app.md

```diff
@@ -79,18 +79,20 @@ The `web` property defines how your application is exposed to the web (in HTTP).
 
 You can specify the following attributes for the `web` property:
 
--  `document_root`: The path relative to the root of the application that is exposed on the web. Typical values include `/public` and `/web`.
--  `passthru`: The URL used in the event a static file or PHP file could not be found. This would typically be your applications front controller, often `/index.php` or `/app.php`.
--  `index_files`: To use a static file (for example, `index.html`) to serve your application. This key expects a collection. For this to work, the static file(s) should be included in your whitelist. For example, to use a file named `index.html` as an index file, your whitelist should include an element that matches the filename, like `- \.html$`.
```

Comment:
`index_files` | Specifies a static file (for example, `index.html`) to serve your application. This key expects a collection. For this to work, the static file(s) should be included in your whitelist. For example, to use a file named `index.html` as an index file, ensure that your whitelist includes an element that matches the filename, like `- \.html$`.